### ISSUE TITLE
What happened to this editor?

### ISSUE THREAD

**Issue by jordwalke** [etexteditor/e#82]

**Comment by JamyGolden** [comment]
+1?

**Comment by ogrishman** [comment]
I hope the author is fine. It's kinda weird. He says he's still working on the project and then everything just ends. I've written email to the author but did not get any response. I think the software is cool and I love using it. It just need improvements. It's sadly that we don't have the source code of the ecore project, otherwise we can continue to work on it.

**Comment by JamyGolden** [comment]
@adamv Have you been in contact with Vlad by any chance?

**Comment by adamv** [comment]
Nope.

**Comment by ghost** [comment]
Any idea what happed ?

**Comment by apennebaker-ni** [comment]
+1 I miss this project

**Comment by grimalschi** [comment]
+1

**Comment by ghost** [comment]
I move to sublime text editor you can even change theme to similar. 

Also even domain change again owner ;) now its some japanese website:

http://e-texteditor.com

**Comment by VictorHaine** [comment]
I'm currently using PhpStorm.. but e-text would be my first choice...

**Comment by jagtesh** [comment]
Keep the tread going in 2016!

I miss e. It was the first source text editor that I bought. It was somewhat working on Linux before the project stopped getting updates.

**Comment by subtleGradient** [comment]
I was excited about trying this again now that I've mostly switched to Windows. I loved the branching history UI. So awesome!

**Comment by Gomesz785** [comment]
> I move to sublime text editor you can even change theme to similar.
> 
> Also even domain change again owner ;) now its some japanese website:
> 
> http://e-texteditor.com

Yes I think it is occupied by some Japanese entity, but I can't see why, perhaps the owner sold it (for what gain?):
* it seems like a very generic website.
* doesn't the users of that company become confused when they see that?

**Comment by JamyGolden** [comment]
@Gomesz785 what makes you think the owner sold it? It seems more likely to me that it expired and someone else got hold of it once it was free.
